block-template-parts/footer.html

```diff
@@ -0,0 +1,39 @@
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:spacer {"height":70} -->
+    <div style="height:70px" aria-hidden="true" class="wp-block-spacer"></div>
+    <!-- /wp:spacer -->
+    
+    <!-- wp:columns {"align":"wide"} -->
+    <div class="wp-block-columns alignwide"><!-- wp:column -->
+    <div class="wp-block-column"><!-- wp:paragraph -->
+    <p><a href="mailto:#">example@example.com<br></a>T. +00 (0)1 22 33 44 55</p>
```

Comment:
I assume once the theme moves into a non-template state, the hardcoded footer content will be replaced by Widgets?